playbooks/de-deploy-services.yaml

```diff
@@ -53,6 +53,23 @@
     - role: util-notify-chat
       msg: ":heavy_check_mark: Done deploying data-info"
 
+- name: Redeploy image-janitor
+  hosts: image-janitor:&systems
+  become: true
+  gather_facts: false
+  tags:
+    - services
+    - image-janitor
+  roles:
+    - role: util-notify-chat
+      msg: "Deploying image-janitor"
+    - role: de-deploy-service
+      deploy_use_color: false
+      has_configs: "{% if use_consul_configs %}{{ false }}{% else %}{{ true }}{% endif %}"
+      service_name: "{{image_janitor.compose_service}}"
+    - role: util-notify-chat
+      msg: ":heavy_check_mark: Done deploying anon-files"
```

Comment:
Should get this line saying the right service name, even if it's just for our amusement in slack.